hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java

```diff
@@ -259,6 +252,68 @@ private InvolvedParties computeInvolvedParties(
         return parties;
     }
 
+    private boolean contractNotRequired(@NonNull final HederaEvmAccount to, @NonNull final Configuration config) {
```

Comment:
`to` should be annotated `@Nullable`, right?

Not sure if there are cases where it would matter, but I think we should also get the number of the account via,
```
final var maybeGrandfatheredNumber = (to == null) ? null : to.hederaId().accountNumOrThrow();

return featureFlags.isAllowCallsToNonContractAccountsEnabled(config, maybeGrandfatheredNumber);
```
since this would work even if the account had an EVM address outside the long-zero subspace.

Reply:
This seems to fail in the case that the `to` address is a token address.